app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,18 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self,

```

Comment:
Make the `__init__` signature like in the `Dwarf` class, so that each argument will start from a new line.